test/e2e/plain_provisioner_test.go

```diff
@@ -829,6 +829,136 @@ var _ = Describe("plain provisioner bundle", func() {
 		})
 	})
 
+	When("the bundle is backed by a non-exist configmap", func() {
+		var (
+			bundle *rukpakv1alpha1.Bundle
+			ctx    context.Context
+		)
+
+		BeforeEach(func() {
+			ctx = context.Background()
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "combo-local-",
+				},
+				Spec: rukpakv1alpha1.BundleSpec{
+					ProvisionerClassName: plain.ProvisionerID,
+					Source: rukpakv1alpha1.BundleSource{
+						Type: rukpakv1alpha1.SourceTypeLocal,
+						Local: &rukpakv1alpha1.LocalSource{
+							ConfigMapRef: &rukpakv1alpha1.ConfigMapRef{
+								Name:      "Non-exist",
+								Namespace: defaultSystemNamespace,
+							},
+						},
+					},
+				},
+			}
+			err := c.Create(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+
+		AfterEach(func() {
+			err := c.Delete(ctx, bundle)
+			Expect(client.IgnoreNotFound(err)).To(BeNil())
+		})
+		It("checks the bundle's phase gets failing", func() {
```

Comment:
nit: eventually results in a failing bundle state?

Reply:
I change it.  Thanks!